embedded/embedded-impl/src/main/java/org/mule/runtime/module/embedded/impl/EmbeddedController.java

```diff
@@ -169,7 +169,7 @@ private void setUpEnvironment() {
     URL[] urLs = urlClassLoader.getURLs();
     for (URL urL : urLs) {
       File dependencyFile = new File(urL.getPath());
-      if (dependencyFile.getName().endsWith("zip")) {
+      if (dependencyFile.getName().toLowerCase().endsWith("zip")) {
```

Comment:
In many places this kind of code is used for dependency names. Would it make sense to have a `Dependency` abstraction or something similar? Maybe not in this PR, but at least to add a TODO.